packages/core/ts/MobileCore.ts

```diff
@@ -145,17 +146,29 @@ const MobileCore: IMobileCore = {
     return RCTAEPCore.getSdkIdentities();
   },
 
+   /**
+   * Called by the extension public API to dispatch an event for other extensions or the internal SDK to consume.
+   * Any events dispatched by this call will not be processed until after `start` has been called.
+   *
+   * @param event Required parameter with {@link Event} instance to be dispatched. Should not be nil
+   * @return true if the the event dispatching operation succeeded, otherwise the promise will return an error
```

Comment:
missing `.`

Reply:
I added the periods.  But in this file not param and return have periods after the sentences.